# Description: Fix empty master.srv creation, which cause segfaults when starting network games
# Forwaded: yes : https://sourceforge.net/tracker/?func=detail&aid=3207635&group_id=110997&atid=657950
# Bug: none
# Author: Christiaan Welvaart (spturtle)
Index: src/network/nServerInfo.cpp
===================================================================
--- src/network/nServerInfo.cpp
+++ src/network/nServerInfo.cpp	2011-03-12 12:01:33.725306330 +0100
@@ -1487,7 +1487,7 @@
     if (!masterInfo)
         return;
 
-    DeleteAll();
+    DeleteAll(false);
 
     // load all the servers we know
     Load( tDirectories::Var(), MasterFile( fileSuffix ) );
